unitex/build.xml

```diff
@@ -63,7 +63,9 @@
 
 	<target name="dist" depends="clean,compile" description="generate the distribution">
 		<!-- create Unitex.jar -->
-		<jar jarfile="${dist-dir}/Unitex.jar" manifest="classes/fr/umlv/unitex/Manifest.mf" basedir="classes" excludes="classes/fr/loria/**" />
+		<jar jarfile="${dist-dir}/Unitex.jar" manifest="classes/fr/umlv/unitex/Manifest.mf" basedir="classes" excludes="classes/fr/loria/**">
+			<zipgroupfileset dir="${lib-dir}" includes="juniversalchardet-1.0.3.jar"></zipgroupfileset>	
```

Comment:
It would be necessary to update the [Windows Installer](https://github.com/UnitexGramLab/unitex-packaging-windows/blob/master/unitex.nsi#L206) to allow the install of juniversalchardet, nowadays .jar dependencies names are hard-coded. This should be easy to solve.